src/commands/game_commands/hint.ts

```diff
@@ -6,19 +6,29 @@ import { KmqImages } from "../../constants";
 import { getGuildPreference } from "../../helpers/game_utils";
 import { GuessModeType } from "../game_options/guessmode";
 import { codeLine } from "../../helpers/utils";
-import { GuildTextableMessage } from "../../types";
+import { GuildTextableMessage, GameType } from "../../types";
 import GameSession from "../../structures/game_session";
+import EliminationScoreboard from "../../structures/elimination_scoreboard";
 
 const logger = _logger("hint");
 
 function isHintMajority(message: GuildTextableMessage, gameSession: GameSession): boolean {
+    if (gameSession.gameType === GameType.ELIMINATION) {
+        const eliminationScoreboard = gameSession.scoreboard as EliminationScoreboard;
+        return gameSession.gameRound.getHintRequests() >= Math.floor(eliminationScoreboard.getAlivePlayersCount() * 0.5) + 1;
+    }
     return gameSession.gameRound.getHintRequests() >= getMajorityCount(message);
 }
 
 async function sendHintNotification(message: GuildTextableMessage, gameSession: GameSession) {
+    let majorityCount = getMajorityCount(message);
```

Comment:
Move this to after the `elimination` condition. Also, be consistent with `isHintMajority.` You aren't storing it in a separate variable there